assets/charts/control-plane/kubelet/templates/kubelet-ds.yaml

```diff
@@ -65,36 +65,50 @@ spec:
         securityContext:
           privileged: true
         volumeMounts:
+        # Used by Calico.
         - mountPath: /var/lib/cni
           name: coreos-var-lib-cni
           readOnly: false
+        # This is in needed so that the calico CNI plugin can read the file named `nodename` in the
+        # following path. This file is created by `calico-node` daemonset pod.
         - mountPath: /var/lib/calico
           name: coreos-var-lib-calico
           readOnly: true
+        # Used by Calico.
```

Comment:
Again, this is generic to all CNI plugins, which will install CNI binaries into host FS (usually via DaemonSet).